ee/desktop/user/client/client.go

```diff
@@ -95,6 +98,35 @@ func (c *client) DetectPresence(reason string, interval time.Duration) (time.Dur
 	return durationSinceLastDetection, detectionErr
 }
 
+func (c *client) CreateSecureEnclaveKey() (*ecdsa.PublicKey, error) {
+	resp, err := c.base.Post("http://unix/secure_enclave_key", "application/json", http.NoBody)
+	if err != nil {
+		return nil, fmt.Errorf("getting secure enclave key: %w", err)
+	}
+
+	if resp.Body == nil {
+		return nil, errors.New("response body is nil")
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("unexpected status code: %d", resp.StatusCode)
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, fmt.Errorf("reading response body: %w", err)
+	}
+
+	key, err := echelper.PublicB64DerToEcdsaKey(body)
+	if err != nil {
+		return nil, fmt.Errorf("converting key: %w", err)
+	}
```

Comment:
This is a nitpick, sorry I didn't notice it sooner --

I think it makes more sense from a separation of responsibilities standpoint to have this client function return the raw bytes of the body, and have this conversion happen inside the runner's `CreateSecureEnclaveKey` func instead -- since the runner implements an interface called `secureEnclaveClient`, it feels appropriate for the runner to know about the key format, but it doesn't seem necessary for the http client here to know about the key format.

Reply:
done